StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/STPFixtures+PaymentSheet.swift

```diff
@@ -127,7 +127,6 @@ extension STPElementsSession {
         externalPaymentMethodTypes: [String] = [],
         customerSessionData: [String: Any]? = nil,
         cardBrandChoiceData: [String: Any]? = nil,
-        isLinkPassthroughModeEnabled: Bool? = nil,
```

Comment:
Removing this parameter, as the `link_mode` determines the passthrough state. Added logic below so that `link_settings.link_passthrough_mode_enabled` is still set correctly.